refactor CreatePartitions endpoint of Kafka API to use TUpdateSchemeActor

siarheivesialou opened the pull request.

siarheivesialou committed:
LOGBROKER-8860: refactor CreatePartitions endpoint of Kafka API to use TUpdateSchemeActor

alexnick88 commented on ydb/core/kafka_proxy/actors/kafka_create_partitions_actor.cpp:
```diff
@@ -214,47 +213,33 @@ class TKafkaCreatePartitionsRequest : public NKikimr::NGRpcService::IRequestOpCt
-                    } else {
-                        SendResultCallback(TEvKafka::TEvTopicModificationResponse::EStatus::ERROR, ReplyMessage);
-                    }
+                if (Issue.IssueCode == Ydb::PersQueue::ErrorCode::ACCESS_DENIED) {
```
Почему такой выбор именно конвертации scheme_error?

siarheivesialou replied:
Обсудили, исправил

siarheivesialou committed:
Fix ydb->kafka status converstion for CreateTopics and CreatePartitions.
Fix incorrect output format in case of two same topics provided for Create Partitions.
Refactorings + style fixes.
